examples/apps/aml/web-client/src/config.js

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2019 Banco Santander S.A.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+export default {
+
+    SERVER_URL : 'http://{YOUR DOMAIN}.com/',
```

Comment:
Refrain from using trailing separators '/'. Sanitize in implementation and use. As I tried to run, I casually removed it as an IP had to be supplied. I spent some build cycles to get over it.

Reply:
How should I modify it?

Reply:
SERVER_URL : '{YOUR SERVER URL}', // eg - http://localohost, http://myaml.com
This would suffice. While using, you could append the trailing separator '/'.